docs/docs/contributing-guide/setup/Mac OS.md

```diff
@@ -32,9 +32,13 @@ Follow these steps to setup and run ToolJet on Mac OS. Open terminal and run the
     gem install bundler:2.1.4
     ```
 
-    ### Install Node.js
+    ### Install Node.js with 14.9.0 version, you can use nvm(Node Version Manager)
```

Comment:
Can we change this line to `Install Node.js ( version: v14.9.0 )`

Reply:
Sure, will change it.

Reply:
@Navaneeth-pk Done with above change.